RecoTracker/TkSeedingLayers/src/SeedingLayerSetsBuilder.cc

```diff
@@ -363,14 +366,61 @@ std::vector<SeedingLayerSetsBuilder::SeedingLayerId> SeedingLayerSetsBuilder::la
 
 std::unique_ptr<SeedingLayerSetsHits> SeedingLayerSetsBuilder::hits(const edm::Event& ev, const edm::EventSetup& es) {
   updateEventSetup(es);
+  
+  auto ret = std::make_unique<SeedingLayerSetsHits>(theNumberOfLayersInSet,
+                                                    &theLayerSetIndices,
+                                                    &theLayerNames,
+                                                    &theLayerDets);
+  
+  for(auto& layer: theLayers) {
+    ret->addHits(layer.nameIndex, layer.extractor->hits((const TkTransientTrackingRecHitBuilder &)(*theTTRHBuilders[layer.nameIndex]), ev, es));
+  }
+  ret->shrink_to_fit();
+  return ret;
+}
+std::unique_ptr<SeedingLayerSetsHits> SeedingLayerSetsBuilder::makeSeedingLayerSetsHits(const edm::Event& ev, const edm::EventSetup& es) {
```

Comment:
Please add `ForFastSim` or similar to the method name to make it clear that it is FastSim specific.

Reply:
Okay. Done.